yt_dlp/extractor/recurbate.py

```diff
@@ -25,7 +25,7 @@ def _real_extract(self, url):
             webpage = self._download_webpage(url, video_id)
         except ExtractorError as e:
             if isinstance(e.cause, urllib.error.HTTPError) and e.cause.code == 403:
-                self.raise_login_required()
+                self.raise_login_required(msg='This video is only available for registered users; Set your authenticated browser user agent via the --user-agent parameter.', method='cookies')
```

Comment:
I don't know if this message is clear enough, but the --user-agent parameter is required to match the one set during the browser session.

Is there also an automated way to do that?

Reply:
> > Is there also an automated way to do that?
> 
> No

Well then, nothing to add.

The extractor works with the `--user-agent` parameter